### ISSUE TITLE
aria-expanded not present with multiple type ahead

### ISSUE THREAD

**Issue by barettjfed** [ericgio/react-bootstrap-typeahead#763]
### Version
6.0.0

### Steps to reproduce

1. Create a basic Typeahead component
2. Notice that the aria-expanded prop is working as expected
3. Add the prop "multiple=true"
4. Now the aria-expanded prop is not present

### Expected Behavior

I expect the aria-expanded props to be present just like the single select. I would also expect the inputProps to be able to pass that aria-expanded down, but it gets overwritten.

### Actual Behavior

aria-expanded prop does not show up when using multiple. Unable to pass that prop down with inputProps as well.

**Comment by ericgio** [comment]
Hey @barettjfed, thanks for opening this issue. I agree the accessibility for the multi-select case needs improvement. As I recall the [spec](https://www.w3.org/TR/wai-aria-1.1/#combobox) was a little ambiguous when it came to multi- vs. single-select comboboxes, but maybe I was just misunderstanding.

Based on other examples, I think the multi-select version should just be more in line with a11y for the single-select version, and additionally add `aria-multiselect=true`

**Comment by ericgio** [comment]
Fixed in v6.0.1
